src/vl-map.js

```diff
@@ -263,7 +285,9 @@ export class VlMap extends vlElement(HTMLElement) {
   }
 
   __initializeCoordinateSystem() {
-    proj4.defs('EPSG:31370', '+proj=lcc +lat_1=51.16666723333333 +lat_2=49.8333339 +lat_0=90 +lon_0=4.367486666666666 +x_0=150000.013 +y_0=5400088.438 +ellps=intl +towgs84=-106.869,52.2978,-103.724,0.3366,-0.457,1.8422,-1.2747 +units=m +no_defs');
+    proj4.defs('EPSG:31370', '+proj=lcc +lat_1=51.16666723333333 +lat_2=49.8333339 +lat_0=90 +lon_0=4.367486666666666 +x_0=150000.013 +y_0=5400088.438 +ellps=intl +towgs84=106.869,-52.2978,103.724,-0.33657,0.456955,-1.84218,1 +units=m +no_defs');
```

Comment:
Waarom deze wijziging?

Reply:
Dit is de definitie van EPSG:31370 van spatialreference.org en dit zou de correcte zijn.